### ISSUE TITLE
Some biomass - carbon fiber related recipes only show one side in JEI

### ISSUE THREAD

**Issue by knzconnor** [FTBTeam/FTB-Modpack-Issues#4489]
### Modpack

FTB Genesis

### Modpack version

1.4.0

### Describe the bug

There are some recipes that are one sides in JEI (show up from the end of the ingredients, but can't be found via the product) in such a way it makes it seem like a gating issue (but Saereth said they are just missing). Specifically carbon dust doesn't show the actual set of machines that will make it, so it seems like it's gated on Mekanism, but even the IC2 macerator will make it if you search from that end.

And I don't think the non-tree biomass/plant ball/whatever show up. The second one just cycles through leaves. I think I waited long enough that this is true. But they definitely show up if you look for what things like berries can make.

Both of these felt like intentional gating of carbon fiber production but apparently are just JEI missed connections?

### Steps to reproduce

1. Open JEI
2. Search for carbon fiber of plant balls
3. Observe what doesn't show up

### Expected behaviour

Recipes that show up when showing use of machines and products also show up when browsing for the recipe for the product

### Additional information

Thanks, great pack.

**Comment by ArtpokeBr** [comment]
Added it to JEI again! if anything else also doesn't show up, please open another issue! =)
